### PR TITLE
feat: Add Flux bootstrap for Gitea

### PR DESCRIPTION
**Pull request by Azhovan**
This PR introduces a page that provides guidance specific to Gitea. 

Related to: https://github.com/fluxcd/flux2/pull/4270 bootstrap gitea command

### CODE REVIEW

**Comment by Azhovan** [comment]
Ping @makkes

**Comment by makkes** [comment]
This will need to be hold until the next Flux minor release is out.

File: content/en/flux/installation/bootstrap/gitea.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+You can also supply the token using a pipe e.g. `echo "<gt-token>" | flux bootstrap gitea`.
+
+## Gitea Personal Account
+
+If you want to bootstrap Flux for a repository owned by a personal account, you can generate a
+[Gitea PAT](https://gitea.com/user/settings/applications)
+that can create repositories by checking all permissions under `Select permissions` drop down menu.
+
+If you want to use an existing repository, the PAT's user must have `admin`
+[permissions](https://docs.gitea.com/development/oauth2-provider#scopes).
```
Comment (makkes) [inline content/en/flux/installation/bootstrap/gitea.md:44]: Please provide more details on how to create the token with this permission. I don't see anything like "admin" in the UI for creating a token.

**Review by stefanprodan — APPROVED** [review]

File: content/en/flux/installation/bootstrap/gitea.md (hunk 1 above)
Comment (stefanprodan) [reply content/en/flux/installation/bootstrap/gitea.md:44]: Look like it's called `all` not admin, https://github.com/fluxcd/go-git-providers/blob/1f17f86ce0f5e1899185b59d5d0347c208ba80ec/tests/gitea/await-healthy.sh#L15C3-L15C3

**Pull request merged**